website/docs/d/dns_custom_resolvers.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+subcategory: "DNS Services"
+layout: "ibm"
+page_title: "IBM : Custom Resolvers"
+description: |-
+  Manages IBM Cloud Infrastructure Private DNS Custom Resolvers.
+---
+
+# ibm_dns_custom_resolvers
+
+Provides a read-only data source for Custom Resolvers. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.For more information about Custom Resolver, refer to [List custom resolvers](https://cloud.ibm.com/apidocs/dns-svcs#list-custom-resolvers).
+
+
+## Example usage
+
+```terraform
+data "ibm_dns_custom_resolvers" "example-cr" {
+  instance_id = "resource_instance_guid"
+}
+```
```

Comment:
ibm_dns_custom_resolvers - is this going to return more then 1?  why plural

Reply:
yes it will return more than 1 custom_resolvers